elementary/monitor/cli.py

```diff
@@ -291,13 +300,16 @@ def send_report(
     anonymous_tracking.track_cli_start('monitor-send-report', get_cli_properties(), ctx.command.name)
     try:
         config.validate_send_report()
+        # If file name contains a folders (is a path),
+        # we want to make sure that we create the report localy without the folders of the full path.
+        local_file_path = os.path.basename(bucket_file_path) if bucket_file_path else slack_file_name
```

Comment:
Same here - it's a not a local folder. I'd suggest doing something like - bucket_file_path.rsplit('/', 1)

Reply:
Still using path.basename